brands/amenity/fast_food.json

```diff
@@ -1285,6 +1297,17 @@
       "takeaway": "yes"
     }
   },
+  "amenity/fast_food|Mucho Burrito": {
+    "countryCodes": ["ca", "gb", "us"],
+    "tags": {
+      "amenity": "fast_food",
+      "brand": "Mucho Burrito",
+      "brand:wikidata": "Q1689380",
```

Comment:
This one too .. Can you please fix the qids and let me know when this is ready for review?

Reply:
fixed